Style Guide and Diataxis Review: Streams IV (#221)
iotaledger/streams: documentation/docs/welcome.md

---

> description: IOTA Streams can be used for the integration of an IOTA Wallet into your application. 

Reviewer: ```suggestion
description: IOTA Streams can be used to integrate an IOTA Wallet into your application. 
```
